setup.py

```diff
@@ -19,10 +19,9 @@
         "Programming Language :: Python :: 3",
         "License :: OSI Approved :: MIT License"
       ],
-      install_requires=['numpy', 'requests', 'pillow', 'tqdm', 'networkx'],
+      install_requires=['numpy', 'requests', 'pillow', 'tqdm', 'networkx', 'pyopencl'],
```

Comment:
Why should this be required? If there's no GPU, fallback to the CPU as it currently does.